libraries/engage/product/components/UnitQuantityPicker/UnitQuantityPicker.jsx

```diff
@@ -4,6 +4,7 @@ import { css } from 'glamor';
 import classNames from 'classnames';
 import { themeConfig } from '@shopgate/engage';
 import { RippleButton, QuantityInput } from '@shopgate/engage/components';
+import { useWidgetSettings } from '../../../core';
```

Comment:
i would prefer here an absolut import like

```suggestion
import { useWidgetSettings } from '@shopgate/engage/core';
```